packages/widgetbook_cli/bin/models/cli_args.g.dart

```diff
@@ -0,0 +1,30 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
+
+part of 'cli_args.dart';
+
+// **************************************************************************
+// JsonSerializableGenerator
+// **************************************************************************
+
+_$_CliArgs _$$_CliArgsFromJson(Map<String, dynamic> json) => _$_CliArgs(
+      apiKey: json['apiKey'] as String,
+      branch: json['branch'] as String,
+      commit: json['commit'] as String,
+      gitProvider: json['gitProvider'] as String,
+      gitHubToken: json['gitHubToken'] as String?,
+      prNumber: json['prNumber'] as String?,
+      baseBranch: json['baseBranch'] as String?,
+      path: json['path'] as String,
+    );
+
+Map<String, dynamic> _$$_CliArgsToJson(_$_CliArgs instance) =>
+    <String, dynamic>{
+      'apiKey': instance.apiKey,
```

Comment:
Do we need this? is this data serialized to a json file?

Reply:
no resolving